src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.theelixrlabs.healthcare.constants;
+
+public class DoctorConstants {
+    public static final String DOCTOR_COLLECTION_NAME = "doctor";
```

Comment:
is collection specific to single doctor ?
if not change it to "doctors"

Reply:
yes changed it to doctors.